app/controllers/mentor_signup_controller.rb

```diff
@@ -164,8 +164,8 @@ def fetch_mentor
   end
   
   def fetch_term
-    @term = params[:term_id].blank? ? Term.current_term : Term.find(params[:term_id])
-    @term ||= Term.allowing_signups.try(:first) || Term.last
+    @term = params[:term_id].blank? ? Term.allowing_signups.try(:first) : Term.find(params[:term_id])
```

Comment:
Right now, when a new mentor goes to uwdp.dreamsis.com and ends up at the mentor_signup/basics page, they see "You can't sign up for a visit, please talk to your HSL" message, even though signups are open for Spring quarter.